lib/linux_admin/security.rb

```diff
@@ -0,0 +1,18 @@
+module LinuxAdmin
+  class Security
+    def self.scap_lockdown
+      SshdConfig.apply_scap_settings
+      Service.apply_scap_settings
+      SysctlConf.apply_scap_settings
+      LimitsConf.apply_scap_settings
+      Securetty.remove_vcs
```

Comment:
Should this method be named `apply_scap_settings`?  Same question for AuditRules.

If they are all named the same, this could be written as:

```ruby
[SshdConfig, Service, SysctlConf, LimitsConf, Securetty, LoginDefs, Useradd, AuditRules, AuditRules, Modprobe].each do |klass|
  klass.public_send(:apply_scap_settings)
end
```

or you can multiline the class array if you prefer.